.eslintrc.json

```diff
@@ -1,5 +1,9 @@
 {
     "extends": "airbnb-base",
+    "env" : {
```

Comment:
why are these needed?

Reply:
`eslint` give errors of the form : 
```
12:3   error  'it' is not defined                                            no-undef
18:3   error  'describe' is not defined                                      no-undef
```

when `mocha` env is not set to be true.